app/src/main/java/de/danoeh/antennapod/activity/DirectoryChooserActivity.java

```diff
@@ -37,6 +43,7 @@
 
 	public static final String RESULT_SELECTED_DIR = "selected_dir";
 	public static final int RESULT_CODE_DIR_SELECTED = 1;
+    public static final String NON_EMPTY_DIRECTORY_WARNING = "warn_non_empty_directory";
```

Comment:
The original code seems to use tabs and spaces whereas my code uses spaces exclusively. In Android Studio it looks completely normal.